sdk/nodejs/errors.ts

```diff
@@ -81,3 +81,28 @@ export function isGrpcError(err: Error): boolean {
     const code = (<any>err).code;
     return code === grpc.status.UNAVAILABLE || code === grpc.status.CANCELLED;
 }
+
+export class InvalidInputDetails {
```

Comment:
`PropertyError` like the protobuf message name?

Reply:
Yeah, I think that's better.  Renamed.